pycbc/filter/matchedfilter.py

```diff
@@ -1849,10 +1875,15 @@ def followup_event_significance(ifo, data_reader, bank,
         dq_start_time = onsource_start - duration / 2.0
         dq_duration = onsource_end - onsource_start + duration
         if not data_reader.dq.is_extent_valid(dq_start_time, dq_duration):
+            logging.info(
+                '%s DQ buffer indicates invalid data during onsource window, '
+                'will not use for followup',
+                ifo
+            )
             return None
 
     # Calculate SNR time series for this duration
-    htilde = bank.get_template(template_id, min_buffer=bdur)
+    htilde = bank.get_template(template_id, td_samples=bdur_samples)
```

Comment:
Would it make more logical sense to just request a specific delta_f?

Reply:
…though the only thing it needs it for is to calculate `flen`, which in principle could be done as `flen = int(sample_rate / delta_f / 2 + 1)`.

Reply:
Ok, I have implemented this comment now.